### PR TITLE
Add php example

### PR DESCRIPTION
**Pull request by withinboredom**
# Description

This adds a "percentage" operation to the distributed calculator using the PHP SDK.

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] The quickstart code compiles correctly
* [x] You've tested new builds of the quickstart if you changed quickstart code
* [x] You've updated the quickstart's README if necessary

### CODE REVIEW

File: distributed-calculator/README.md [hunk 1]
```diff
@@ -95,7 +96,16 @@ These instructions start the four calculator operator apps (add, subtract, multi
       ```
       dapr run --app-id multiplyapp --app-port 5000 --dapr-http-port 3501 flask run
       ```
-5. Frontend Calculator app - Open a terminal window and navigate to the react-calculator directory and follow the steps below:
+5. Percentage App - Open a terminal window and navigate to the php directory and follow the steps below:
+   - Install the required packages and PHP 8
+     ```
+     composer install -o --no-dev
```
Comment (artursouza) [inline distributed-calculator/README.md:102]: How do I install composer? Add install step.

File: distributed-calculator/README.md [hunk 2]
```diff
@@ -95,7 +96,16 @@ These instructions start the four calculator operator apps (add, subtract, multi
...
+     ```
+   - Start dapr using the command:
+     ```
+     dapr run --app-id percentageapp --app-port 3005 --dapr-http-port 3505 -- php -S 0.0.0.0:3005
+     ```
+6. Frontend Calculator app - Open a terminal window and navigate to the react-calculator directory and follow the steps below:
```
Comment (artursouza) [inline distributed-calculator/README.md:108]: Update the numbers of the following steps.

**Review by artursouza — COMMENTED** [review]
Please, validate using minikube too.

**Comment by orizohar** [comment]
I think the distributed calculator is a bit of a problematic quickstart in general since even today it has too many pre-requisite steps (I'm actually for considering to retire it or replace it with something simpler). It was initially intended to show a "polyglot app" that uses several languages but I'm not sure if adding another language adds value here aside from another dependency.

For future reference - in general we try to have PRs attached to issues which is where we discuss proposals before people spend the time to implement (as mentioned in the [contribution guide](https://docs.dapr.io/contributing/contributing-overview/#pull-requests)). Specifically, the quickstart repo is a bit more controlled as it is intended for new users and should remain as simple as we can make it.

**Comment by yaron2** [comment]
I think this should go into the samples repo.

/cc @withinboredom @orizohar

**Comment by withinboredom** [comment]
Sounds good!

**Pull request closed without merging**